src/views/protected/Profile.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="box " style="text-align: center">
+            <ProfileComponent></ProfileComponent>
+          <div class="popup" >
```

Comment:
please format the file properly to prevent such bogus newlines and spaces. This is applicable for all the files in the PR @raju-max